core/templates/services/literally-canvas-helper.service.ts

```diff
@@ -58,17 +58,20 @@ angular.module('oppia').factory('LiterallyCanvasHelperService', [
       }
     };
 
-    var isSvgTagValid = function(svgString) {
+    var _isSvgTagValid = function(svgString) {
       // This function validates an svg tag.
       var dataURI = 'data:image/svg+xml;base64,' + btoa(svgString);
       var invlidTagsAndAttr = (
         ImageUploadHelperService.getInvalidSvgTagsAndAttrs(dataURI));
-      if (invlidTagsAndAttr.tags.length === 0 &&
-          invlidTagsAndAttr.attrs.length === 0) {
-        return true;
-      } else {
-        throw new Error('Invalid tag or attribute in svg.');
+      if (invlidTagsAndAttr.tags.length !== 0) {
```

Comment:
Misspelled "invalid", here and elsewhere. Please proofread your work carefully.

Reply:
Done.